tests/amplitude/test_yaml_helicity.py

```diff
@@ -105,22 +107,22 @@ def test_parameter_section(self):
 
     def test_dynamics_section(self):
         dynamics = self.imported_dict["Dynamics"]
-        assert len(dynamics) == 5
+        assert len(dynamics) == 1
```

Comment:
This change is correct? I did not verify in the recipe. Just want to make sure this is supposed to change

Reply:
This is because this PR does the following:
`particle_list.xml` --> `DATABASE` [nested dict]
becomes
`particle_list.xml` -> `ParticleCollection`  --> `DATABASE`  [nested `dict`]
Since dynamics are not retained in the `ParticleCollection`, this information is lost, so the dynamics section of the recipe file shrinks.

Reply:
So how is the the dynamics added to the final recipe yml then?

Reply:
https://github.com/ComPWA/expertsystem/pull/110#issuecomment-656055278

Reply:
Seems like i need to dig a bit deeper into these code changes

Reply:
But then i expect this thing to be empty? But somehow the `J/Psi` in this example is in there

Reply:
Yeah because initial state is always given non-dynamic. Not sure where, I can look it up

Reply:
That is added in the helicity amplitude generator. Then we should also remove this part

Reply:
all right as it triggers another bigger change lets move this to a follow up pr